### ISSUE TITLE
False positive: eslint-config-eslint:all

### ISSUE THREAD

**Issue by sfuerte** [depcheck/depcheck#404]
```
> depcheck
Missing dependencies
* eslint-config-eslint:all
```

Have the following in `package.json`:
```
...
    "eslintConfig": {
        "env": {
            "node": true,
            "es6": true
        },
        "extends": "eslint:all",
...
    "devDependencies": {
        "eslint": "^6.1.0",
...
```

### Versions
 - `node -v`: v12.7.0
 - `npm -v`: 6.10.0
 - `depcheck --version`: 0.8.2

**Comment by KaboomFox** [comment]
This should be fixed in the next release

**Comment by VincentLanglet** [comment]
@rumpl This can be closed

**Comment by rumpl** [comment]
Woop woop
